.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+    "printWidth": 90,
+    "tabWidth": 4,
+    "useTabs": false,
+    "semi": true,
+    "singleQuote": false,
+    "trailingComma": "none",
```

Comment:
I actually think trailing commas could be nice - what are your thoughts on setting this to `es5`? It would be nice to avoid having to always to add a comma to an array or object every time somthing new is added.

It would also clean up diffs and help avoid merge conflicts in those areas, because you'd only be editing a single line instead of two.